pkg/dsdk/connection.go

```diff
@@ -242,13 +310,13 @@ func (c *ApiConnection) do(ctxt context.Context, method, url string, ro *greq.Re
 	if _, ok := ctxt.Value("quiet").(bool); ok {
 		rdata = "<muted>"
 	}
-	Log().Debugf(strings.Join([]string{"\nDatera Trace ID: %s",
-		"Datera Response ID: %s",
-		"Datera Response TimeDelta: %fs",
-		"Datera Response URL: %s",
-		"Datera Response Payload: %s",
-		"Datera Response Object: %s\n"}, "\n"),
-		tid, reqId, tDelta.Seconds(), gurl.String(), rdata, "nil")
+	Log().WithFields(log.Fields{
+		logTraceID:            tid,
```

Comment:
Any reason this is a constant and the other fields are not?

Reply:
it was used in multiple places